build/environment.yml

```diff
@@ -20,7 +20,7 @@ dependencies:
   - pip:
     - cffi==1.15.0
     - errorhandler==2.0.1
-    - git+https://github.com/manubot/manubot@82f25cb8d698a12d69e533686867a8a4afd35c4c
+    - git+https://github.com/manubot/manubot@d4242ffa4194e4a13a68c5f6466feff559d3f9d5
```

Comment:
What's the reason to install manubot using a changeset hash? If we used a version (`pip install manubot==0.5.5`), it would simplify things when installing `manubot[ai-rev]`

Reply:
The commit hash allows us to upgrade to unreleased manubot versions. Would be somewhat of a hassle if we always had to create manubot releases to upgrade rootstock.

Does adding `#egg=manubot[ai-rev]` to the git dependency not work?